internal/dbsqlc/river_leader.sql

```diff
@@ -40,7 +40,7 @@ WITH currently_held_leaders AS (
 ),
 notified_resignations AS (
   SELECT
-    pg_notify('river_leaderhip', json_build_object('name', name, 'leader_id', leader_id, 'action', 'resigned')::text),
+    pg_notify('river_leadership', json_build_object('name', name, 'leader_id', leader_id, 'action', 'resigned')::text),
```

Comment:
Actually, another way to fix this would be to pass in the notification name using the in-code constant. That would ensure we're using the same value everywhere.

Thoughts?

Reply:
Yeah, good idea. It's already referenced in code anyway, so couldn't hurt just to use the same ref here.

Reply:
I’ll do this in a follow up PR, there are a couple we can get rid of